src/utils/web3.ts

```
@@ -1,133 +1,27 @@
-  [C.bsc]: 'https://opensea.io/assets/bsc',
-  [C.testbsc]: 'https://testnets.opensea.io/assets/bsc-testnet',
-  [C.optimism]: 'https://opensea.io/assets/optimism',
-  [C.optimismGoerli]: 'https://testnets.opensea.io/assets/optimism-goerli',
-} as const
-
-declare global {
-  type ChainIds = keyof typeof CHAIN_NAMES
-  type ChainNames = keyof typeof CHAIN_IDS
-  type SupportedChainIds = (typeof SUPPORTED_CHAIN_IDS)[number]
-  type SupportedMainnetChainIds = (typeof SUPPORTED_MAINNET_CHAIN_IDS)[number]
-  type SupportedTestnetChainIds = (typeof SUPPORTED_TESTNET_CHAIN_IDS)[number]
-  type SupportedChainNames = (typeof SUPPORTED_CHAIN_NAMES)[number]
-  type SupportedNetworkNames = (typeof SUPPORTED_NETWORK_NAMES)[number]
-  type DropdownChainIds = (typeof DROPDOWN_CHAIN_IDS)[number]
-  type DropdownChainNames = (typeof DROPDOWN_CHAIN_NAMES)[number]
-  type DisabledChainIds = (typeof DISABLED_CHAIN_IDS)[number]
-}
-
-export const DEFAULT_CHAIN_ID = IS_MAINNET ? C.ethereum : C.goerli
-export const DEFAULT_CHAIN_NAME = CHAIN_NAMES[DEFAULT_CHAIN_ID]
-export const SUPPORTED_MAINNET_CHAIN_IDS = [C.ethereum, C.polygon, C.avalanche, C.bsc, C.optimism]
-export const SUPPORTED_TESTNET_CHAIN_IDS = [C.goerli, C.mumbai, C.fuji, C.testbsc, C.optimismGoerli]
-
-export const SUPPORTED_CHAIN_IDS: Array<SupportedMainnetChainIds | SupportedTestnetChainIds> = IS_MAINNET
-  ? SUPPORTED_MAINNET_CHAIN_IDS
-  : SUPPORTED_TESTNET_CHAIN_IDS
-export const SUPPORTED_CHAIN_NAMES = SUPPORTED_CHAIN_IDS.map(id => CHAIN_NAMES[id])
-export const SUPPORTED_NETWORK_NAMES = SUPPORTED_CHAIN_IDS.map(id => CHAIN_NETWORKS[id])
-export const DISABLED_CHAIN_IDS = []
-export const DISABLED_CHAIN_NAMES = DISABLED_CHAIN_IDS.map(id => CHAIN_NAMES[id])
-
-export const DROPDOWN_CHAIN_IDS = [...SUPPORTED_CHAIN_IDS, C.disconnect_action]
-export const DROPDOWN_CHAIN_NAMES = DROPDOWN_CHAIN_IDS.map(id => CHAIN_NAMES[id])
+
+export const networkToChainId: Readonly<Record<string, number>> = Object.fromEntries(
```

This is a future thing, but i feel like `networkToChainId`, `chainIdtoNetwork` and `NETWORK_COLORS` should be in the networks package.

---

@alexanderattar Oh and when we do move it, I think we should throw an error if someone tries to access a key that does not exist.

---

yeah that's a good idea